controller_interface/include/controller_interface/controller_interface.hpp

```diff
@@ -80,16 +79,30 @@ class ControllerInterface : public rclcpp_lifecycle::node_interfaces::LifecycleN
   void release_interfaces();
 
   CONTROLLER_INTERFACE_PUBLIC
-  virtual LifecycleNodeInterface::CallbackReturn on_init() = 0;
+  virtual return_type init(const std::string & controller_name);
 
+  /// Custom configure method to read additional parameters for controller-nodes
+  /*
+   * Override default implementation for configure of LifecycleNode to get 'update_rate' parameter.
```

Comment:
```suggestion
   * Override default implementation for configure of LifecycleNode to get parameters.
```
let's not describe the entire implementation in the docs, that can get out of date very quickly